core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -151,6 +152,14 @@ class KafkaController(val config: KafkaConfig, zkClient: KafkaZkClient, time: Ti
    */
   def isActive: Boolean = activeControllerId == config.brokerId
 
+  /**
+   * Returns true if the given epoch is the current broker epoch or unknown broker epoch.
+   * Broker epoch in the control request is unknown if the controller hasn't been upgraded to use KIP-380
+   */
+  def isCurrentOrUnknownBrokerEpoch(epoch: Long): Boolean = epoch == AbstractControlRequest.UNKNOWN_BROKER_EPOCH || epoch == brokerEpoch
```

Comment:
I have renamed it and moved this helper function to `KafkaApis` because it will only be called in `KafkaApis` and we will need to throw `IllegalStateException` when the epoch is larger than the expected one. I think it is more readable this way.